### PR TITLE
Add route, actions and view files

### PR DESCRIPTION
**Pull request by SamTush**
## What does this PR do?

This PR is raised to implement the following features: 👍 :

- [x] Create routes
- [x] Create actions 
- [x] Create view files

## Description of the task to be completed ↓↓ 

- Create routes, actions, and views

#### Thanks for reviewing my PR 😊

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -7,5 +7,5 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   resources :users
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#show"
```
Comment (VICTOR-CHUKWU) [inline config/routes.rb]: - [ ] you have to change your routing, your post should be routed to user/:userid/posts and so, your writing should look similar to this 
```
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @SamTush  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SamTush** [comment]
Thank you @VICTOR-CHUKWU for reviewing my PR😎 Cheers and happy coding 🥂

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @SamTush, 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done updating the routes as requested by the previous reviewer. ✔
- You successfully completed all the requirements.  🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SamTush** [comment]
Thank you @ridwanediallo  for reviewing my PR😎 Cheers and happy coding 🥂

**Pull request merged**